app/models/menu.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class Menu < ApplicationRecord
+  has_many :cart_items
+
+  enum category: %i[First Main Drinks]
```

Comment:
```
  enum kind: {
    regular: REGULAR = 'regular'.freeze,
    financed: FINANCED = 'financed'.freeze,
  }
```
this is preferable way, at least in our team.
this way you will have meaningful string instead of 0, 1, 2, 3 in db and that will save you A LOT of trouble later